src/identity/user.service.js

```diff
@@ -116,7 +118,7 @@ function clbUser(
    * provided list of IDs.
    * @function get
    * @memberof module:clb-identity.clbUser
```

Comment:
get should retrieve a single result if no array has been passed as input. This preserve backwards compatibility as the method was not accepting single element before. It is more obvious how to use the service with this addition.